src/components/ec-full-screen-overlay/ec-full-screen-overlay.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <transition name="ec-full-screen-overlay__fade">
+    <div
+      v-show="show"
+      class="ec-full-screen-overlay"
+      data-test="ec-full-screen-overlay"
+    >
+      <div
+        ref="overlayContainer"
+        class="tw-col-full"
+        data-test="ec-full-screen-overlay__container"
+      >
+        <div class="tw-grid tw-grid-cols-3">
+          <div class="tw-col-full sm:tw-col-3" />
+          <div class="tw-col-full sm:tw-col-6">
+            <div class="tw-grid">
+              <div class="tw-col-full sm:tw-col-9">
+                <h1
+                  v-if="title"
+                  data-test="ec-full-screen-overlay__title"
+                  class="ec-full-screen-overlay__title"
+                >
+                  {{ title }}
+                </h1>
+              </div>
+              <div class="tw-col-full sm:tw-col-3 tw-text-right">
+                <a
+                  href="#"
+                  data-test="ec-full-screen-overlay__close-icon-container"
+                  @click="emit('closeOverlay')"
+                >
+                  <ec-icon
+                    class="ec-full-screen-overlay__close-icon"
+                    data-test="ec-full-screen-overlay__close-icon"
+                    name="simple-close"
+                    type="interactive"
+                    :size="15"
```

Comment:
Please speak to designers. I know the design says 15px but is this is wrong for 2 reasons:
* is not a multiple of 4
* Is not big enough, will be hard for people to notice, click, "touch"

Most likely needs to be 24px. Same like ec-modal